lib/query/querycompiler.js

```diff
@@ -892,6 +892,34 @@ class QueryCompiler {
         this.bindingsHolder
       )})`;
     } else {
+      function chunk(arr, start, amount) {
+        var result = [],
+          i,
+          start = start || 0,
+          amount = amount || 500,
+          len = arr.length;
+
+        do {
+          //console.log('appending ', start, '-', start + amount, 'of ', len, '.');
+          result.push(arr.slice(start, start + amount));
+          start += amount;
+        } while (start < len);
+
+        return result;
+    }
+
+    if (Array.isArray(statement.column))
+        return this.multiWhereIn(statement);
+
+    if(statement.value.length > 1000) {
+        let values = chunk(statement.value, 0, 1000);
```

Comment:
Lodash already have a chunk function that do that:
https://www.geeksforgeeks.org/lodash-_-chunk-method/